### DISCUSSION TITLE
How to put 2 sliders on the same row

### DISCUSSION THREAD

**Discussion by nghia-vo** [zauberzeug/nicegui#1779] [Q&A] [answered]
### Question

I tried to do the following, but can't put these two sliders and their labels on the same row:

```python
from nicegui import ui

with ui.row().classes("w-full justify-between"):    
    ui.label("Min: ")
    min_slider = ui.slider(min=0, max=254, value=0).props('label-always')
    ui.label("Max: ")
    max_slider = ui.slider(min=1, max=255, value=255).props('label-always')

ui.run()

```
![image](https://github.com/zauberzeug/nicegui/assets/12136997/4e753153-f8cc-4649-9956-649e4baae328)

**Comment by falkoschindler** [comment, marked as answer]
Hi @nghia-vo!
The problem is that `ui.row` wraps by default if the content gets to wide. You can prevent that using the "no-wrap" class:

```py
with ui.row().classes("w-full justify-between no-wrap"):
    ui.label("Min: ")
    min_slider = ui.slider(min=0, max=254, value=0).props('label-always')
    ui.label("Max: ")
    max_slider = ui.slider(min=1, max=255, value=255).props('label-always')
```

**Reply by nghia-vo** [reply]
Great! It looks much better. Thanks very much. Maybe in the docs of ui.row: https://nicegui.io/documentation/row, the link to Quasar can be pointed to this link: https://quasar.dev/layout/grid/introduction-to-flexbox instead of https://quasar.dev/, so users can have a list of classes to play with. This can be applied to docs of other Nicegui components.

**Reply by nghia-vo** [reply]
Related how to link between nicegui docs and quasar docs. I tried to align two components: ui.label and ui.select using docs from Quasar: https://quasar.dev/style/positioning
```python
from nicegui import ui

with ui.row():
    ui.label("Plot style").style("vertical: middle")
    ui.select(["0", "1", "2"], value="0").style("vertical: top")

with ui.row():
    ui.label("Plot style").classes("vertical-middle")
    ui.select(["0", "1", "2"], value="0").classes("vertical-top")

ui.run()
```
but end with no-effect results (it would look nicer if "plot style" and "0" aligned):
![image](https://github.com/zauberzeug/nicegui/assets/12136997/49b53361-f387-4fe5-820e-c4f806cc24fe)
I can open a new topic if need to.

**Reply by falkoschindler** [reply]
I don't know Quasar's positioning classes very well. I usually use the Tailwind class "items-center". Their [documentation](https://tailwindcss.com/docs/align-items) has pretty helpful illustrations.

**Reply by nghia-vo** [reply]
Thanks for the tip. Work great. Post here in case someone doesn't know which one (classes, style or props) to use:
```python
from nicegui import ui

with ui.row().classes("items-center"):
    ui.label("Plot style")
    ui.select(["0", "1", "2"], value="0")

ui.run()
```
